backend/app.js

```diff
@@ -18,11 +18,13 @@ const app = express();
 app.use(logger("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(
-  bearerToken({
-    headerKey: "Bearer",
-  })
-);
+if (process.env.ZU_DISABLE_AUTH === "true") {
```

Comment:
Hey, @9p4.
Why did you disable the authorization header? That would break the standard authorization flow.

Reply:
Whoops, should be fixed now.

Reply:
Thanks for the fast response! That makes sense now.